packages/vuetify/build/config.js

```diff
@@ -6,6 +6,7 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin')
 
 const base = require('./webpack.prod.config')
 const version = process.env.VERSION || require('../package.json').version
+const dirPostfix = process.env.VUETIFY_DIRECTION ? `.${process.env.VUETIFY_DIRECTION}` : ''
```

Comment:
maybe `-rtl`/`-ltr`?
```suggestion
const dirPostfix = process.env.VUETIFY_DIRECTION ? `-${process.env.VUETIFY_DIRECTION}` : ''
```

Reply:
I think dot is a common convention, not attached to it though, if more people want dash i'm ok. Resolving for now